### DISCUSSION TITLE
Trouble Pruning Optimism mainnet

### DISCUSSION THREAD

**Discussion by pikansj0s** [ethereum-optimism/developers#197] [Node Operators]
Hi! I am having trouble pruning optimism. Can somebody help?

This is the error: ERROR[01-11|11:22:11.523] Failed to create sealing context         err="missing trie node 5cede9f6d48c85df70e129690597b48acda119ab38c2321a29d9c627e2b56859 (path ) state 0x5cede9f6d48c85df70e129690597b48acda119ab38c2321a29d9c627e2b56859 is not available, not found"
ERROR[01-11|11:22:11.523] Failed to build payload                  err="missing trie node 5cede9f6d48c85df70e129690597b48acda119ab38c2321a29d9c627e2b56859 (path ) state 0x5cede9f6d48c85df70e129690597b48acda119ab38c2321a29d9c627e2b56859 is not available, not found"
WARN [01-11|11:22:11.523] Served engine_forkchoiceUpdatedV2        reqid=25954222 duration="833.799µs" err="Invalid payload attributes" errdata="{Error:missing trie node 5cede9f6d48c85df70e129690597b48acda119ab38c2321a29d9c627e2b56859 (path ) state 0x5cede9f6d48c85df70e129690597b48acda119ab38c2321a29d9c627e2b56859 is not available, not found}"

Thanks

**Comment by sbvegan** [comment]
Oops, I didn't check the discussions before I converted this issue. Closing in favor of the duplicate here: https://github.com/ethereum-optimism/developers/discussions/185
